sql/create_tables.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS events (
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
+    venue VARCHAR(255) NOT NULL,
+    date DATE NOT NULL,
+    time TIME NOT NULL,
+    description TEXT,
+    tags VARCHAR(255),
+    imagepath VARCHAR(255),
```

Comment:
We have to figure out how images will be handled. Maybe can look at it in later changes. Fine for now!